_docs/api.md

```diff
@@ -1,9 +1,12 @@
 ---
 layout: documentation
 title: API reference
+description: This page provides an overview of the public The Lounge API.
```

Comment:
I think this is similar to how all my guides started with "How to...". I think here you could go with "Overview of the public API for The Lounge".
Then below remove "This page...", e.g. "Install and upgrade The Lounge" etc. seems sufficient to me.